src/videojs-media-sources.js

```diff
@@ -301,9 +365,9 @@
           offset += segment.byteLength;
         });
 
-        // Set timestampOffset if we have been given one
-        if (this.timestampOffset !== undefined) {
-          destinationBuffer.timestampOffset = this.timestampOffset;
+        // Set timestampOffset if it was changed before this append
```

Comment:
We decided to toss the transmuxer and recreate it whenever we need to [reset the base media decode time](https://github.com/imbcmdth/videojs-contrib-media-sources/commit/1eb75f409a1ce12d25030c8a052244dff5fd7130).